_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -28,31 +28,29 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [WAI-ARIA 1.2 state or property](https://www.w3.org/TR/wai-aria-1.2/#state_prop_def) that is not empty (""), and that is specified on an [HTML or SVG element][].
+This rule applies to any [WAI-ARIA state or property][] that has a non-empty ("") [attribute value][], and that is specified on an [HTML or SVG element][].
 
 ## Expectation
 
-Each test target has a valid value according to its [WAI-ARIA 1.2 value type](https://www.w3.org/TR/wai-aria-1.2/#propcharacteristic_value).
+Each test target has an [attribute value][] that is valid according to its [WAI-ARIA value type][value type], except if one of the following is true:
 
-For value types `ID Reference` and `ID Reference List` for [WAI-ARIA required properties](https://www.w3.org/TR/wai-aria-1.2/#requiredState) at least one of the elements with the given ids exists in the same [document tree](https://dom.spec.whatwg.org/#document-trees) or in the same [shadow tree](https://dom.spec.whatwg.org/#shadow-trees) as the element that specifies the target attribute.
+- <dfn id="off6ek:id-reference">ID Reference</dfn>: For `ID Reference` value types an ID referenced elements is only required for `aria-controls` with [semantic][semantic role] `scrollbar` elements, and with [semantic][semantic role] `combobox` elements that have an `aria-expanded` [attribute value][] of `true`. The ID referenced element must exist in the same [document tree][] or [shadow tree][] as the test target's element.
 
-For value type `URI` the value matches the [generic URI syntax](https://www.ietf.org/rfc/rfc3986.txt).
+- <dfn id="off6ek:id-reference-list">ID Reference List</dfn>: For `ID Reference List` value types, no ID referenced elements are required.
```

Comment:
I feel the exceptions are getting somewhat out of hand and hard to read. Trying something in a different direction to see where it goes…

> For each test target, depending on its type, either:
> - the type is neither ID reference, nor ID reference list, and the value is valid; or
> - the type is ID reference, and the attribute is required, and this is not `aria-controls` on a `combobox` with `aria-expanded != true`, and the value is an existing `id`; or 
> - all other cases pass.

(plus adding Background note that currently no ID reference list is required, and that apart from combobox, the only required ID reference is `aria-controls` on a `scrollbar`)

I prefer listing the combobox as an exception as it makes the rule more future proof (if a new required ID ref comes up, we don't need to change the rule)

---
Maybe it can be make clearer by splitting the requirements:

> Expectation 1
> For each test target, either the type is ID reference/ID reference list; or the value is valid.
>
> Expectation 2
> For each test target, either the the type is not ID reference, or it is not required, or it is `aria-controls` on an closed `combobox`, or the value is an existing `id`.

(again with Background note to explain the reasoning and effect)
This one might makes it harder to see when Expectation 2 triggers since it gets all the negation 🤔